Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Call.hs

```diff
@@ -104,7 +113,41 @@ initiateCallToCustomer rideId = do
         SCS.CallStatus
           { id = callStatusId,
             callId = exotelResponse.callId,
-            rideId = rideId,
+            rideId = Just rideId,
+            rcId = Nothing,
```

Comment:
Rename `rcId` as `entityId` and remove `rideId`.
`entityId` can be rcId or rideId according to usecase.